src/common.jl

```diff
@@ -0,0 +1,51 @@
+abstract DASSLDAEAlgorithm <: AbstractODEAlgorithm
+immutable dassl <: DASSLDAEAlgorithm
```

Comment:
This has come up in many of the other packages. The real problem is, you want to name it `DASSL()`, but you can't because that's the module name. But with the other wrapper packages, we've gone with naming it in lower-case. This convention makes sense for `dopri5()`, and then is easy to adopt like `lsoda()`, `daskr()`. It also makes it so that way `ode23()` is the algorithm in ODE.jl. So I think it's a decent convention, but see where the `Dassl()` idea would come from.